docs/source/css/qb1-sphinx-rtd.css

```diff
@@ -475,6 +475,4 @@ footer span.commit .rst-content tt,
     display: block;
     content: ' ';
     width: 1px;
-    height: 60px;
-    margin-top: -80px;
 }
```

Comment:
I imagine the best way to fix this is to make both height and margin-top to have the same magnitude, e.g. 60px and -60px, or 80px and -80px, so that they cancel each other out.

~I'm trying to figure out what this bit of CSS is supposed to do. My guess is that it's intended to force the target element to appear underneath the fixed nav bar rather than be hidden behind it.~ d'oh, there's a comment that says exactly what it's supposed to do.

I'll see if I can find a solution that accomplishes this goal without breaking anything.